src/site/apt/index.apt

```diff
@@ -16,8 +16,9 @@
 Apache VXQuery
 
   Apache VXQuery\x99 will be a standards compliant XML Query processor implemented in Java.
-  The focus is on the evaluation of queries on large amounts of XML data.
-  Specifically the goal is to evaluate queries on large collections of relatively small XML documents.
+  The XQuery processor supports JSONiq extension to XQuery.
```

Comment:
I'd call it "the JSONiq extension to XQuery" and I think that we should have a link to jsoniq.org somewhere (maybe here, but not necessarily).